src/index.js

```diff
@@ -144,7 +145,13 @@ export default class Video extends WdioReporter {
       return;
     }
 
-    this.addFrame();
+    if (config.skipAlertScreenshots) {
+      browser.isAlertOpen().then((result) => {
+        if (!result) this.addFrame();
+      })
+    } else {
```

Comment:
I wonder if there is a scenario where someone would like to make a screenshot if an alert is open because afaik it would always result in an error.

Reply:
Ok. I've removed the extra variable and added logs.